static/app/components/replays/breadcrumbs/gridlines.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import * as Timeline from 'sentry/components/replays/breadcrumbs/timeline';
+
+import {countColumns, formatTime} from '../utils';
+
+type LineStyle = 'dotted' | 'solid' | 'none';
+
+const Line = styled(Timeline.Col)<{lineStyle: LineStyle}>`
+  border-right: 1px ${p => p.lineStyle} ${p => p.theme.gray100};
+  text-align: right;
+`;
+
+function Gridlines({
```

Comment:
Previously was called `<Ticks>`


Created concrete `<MajorGridlines>` and `<MinorGridlines>` in this file to remove the need for the `showTimestamp` prop.